src/components/layout.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Layout } from 'antd'
+import Navbar from './Navbar'
+import Footers from './Footer'
+import { LayoutCointainer } from './layout.style'
+import 'antd/dist/antd.css'
+
+const { Header, Content } = Layout
+
+const layout = ({ children }) => (

```

Comment:
const Layout

Reply:
if using Layout then it will give error on **Children** prop Error

Reply:
Layout = ({ children })
**'children' is missing in props validation**
so i use
```suggestion
const layout = ({ children }) => (

```